### PR TITLE
Fix Cropping3D documentation

### PR DESCRIPTION
**Pull request by georgwiese**
Found a copy-paste error in the `Cropping3d` documentation.

### CODE REVIEW

File: keras/layers/convolutional.py [hunk 1]
```diff
@@ -1903,14 +1903,14 @@ class Cropping3D(Layer):
     """Cropping layer for 3D data (e.g. spatial or spatio-temporal).
 
     # Arguments
-        cropping: int, or tuple of 2 ints, or tuple of 2 tuples of 2 ints.
+        cropping: int, or tuple of 3 ints, or tuple of 3 tuples of 2 ints.
             - If int: the same symmetric cropping
-                is applied to width and height.
-            - If tuple of 2 ints:
+                is applied to width, height, and depth.
```
Comment (fchollet) [inline keras/layers/convolutional.py]: It's depth, height, and width.

File: keras/layers/convolutional.py [hunk 2]
```diff
@@ -1903,14 +1903,14 @@ class Cropping3D(Layer):
...
+            - If tuple of 3 ints:
                 interpreted as two different
-                symmetric cropping values for height and width:
+                symmetric cropping values for height, width, and depth:
```
Comment (fchollet) [inline keras/layers/convolutional.py]: It's depth, height, and width.

File: keras/layers/convolutional.py (hunk 1 above)
Comment (georgwiese) [inline keras/layers/convolutional.py]: Fixed.

File: keras/layers/convolutional.py (hunk 2 above)
Comment (georgwiese) [inline keras/layers/convolutional.py]: Fixed.

**Pull request merged**